src/composables/trade/useTrading.ts

```diff
@@ -142,7 +141,9 @@ export default function useTrading(
     isBalancerTrade.value ? sor.poolsLoading.value : gnosis.updatingQuotes.value
   );
 
-  const isTrading = computed(() => sor.trading.value || gnosis.trading.value);
+  const isConfirming = computed(
+    () => sor.confirming.value || gnosis.confirming.value
```

Comment:
`isConfirming` is used when a user is waiting to confirm the trade with his wallet (the previous "trading" status is not used for the new trade interface)